miniapps/nurbs/nurbs_naca_cmesh.cpp

```diff
@@ -97,7 +97,7 @@ unique_ptr<KnotVector> TanhKnotVector(int order, int ncp, real_t c);
 // Function that returns a knot vector with a hyperbolic tangent spacing from
 // both sides of the knot vector with a cut-off @c using the @a order and the
 // number of control points @a ncp.
-unique_ptr<KnotVector> DoubleTanhKnotVector(int order, int ncp, real_t c);
```

Comment:
I think `Double` has a double-meaning here and should be kept.

Reply:
Changed back.